<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortView</class>
 <widget class="QWidget" name="SortView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGraphicsView" name="columnView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="swapLabel">
     <property name="text">
      <string>Swaps</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="swapsAmountLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="comparisonsLabel">
     <property name="text">
      <string>Comparisons</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="compAmountLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
